components/eventDetails/MapSnippet.tsx

```diff
@@ -0,0 +1,100 @@
+import { LocationCoordinate2D } from "lib/location/Location"
+import React from "react"
+import { StyleProp, ViewStyle } from "react-native"
+import MapView, { Marker, PROVIDER_GOOGLE } from "react-native-maps"
+
+/**
+ * A type that can be rendered by a {@link EventsMap}.
+ */
+export interface EventMarker {
+  key: string
+  location: LocationCoordinate2D
+}
+
+export type MapProps<T extends EventMarker> = {
+  style?: StyleProp<ViewStyle>
+
+  /**
+   * The initial region to center the map on.
+   */
+  initialRegion: {
+    latitude: number
+    longitude: number
+    latitudeDelta: number
+    longitudeDelta: number
+  }
+
+  /**
+   * A single map marker to display.
+   */
+  marker: T
+
+  /**
```

Comment:
Remove these from the mapSnippet, redundant since settings are static

Reply:
Yes, we can make this component as small as possible, no need to go generic here